exodus_lambda/functions/origin_request.py

```diff
@@ -24,6 +26,57 @@ def __init__(self, conf_file=CONF_FILE):
             timer=time.monotonic,
         )
 
+    @property
+    def db_client(self):
+        if not self._db_client:
+            self._db_client = boto3.client("dynamodb", region_name=self.region)
+
+        return self._db_client
+
+    @property
+    def sm_client(self):
+        if not self._sm_client:
+            self._sm_client = boto3.client(
+                "secretsmanager", region_name=self.region
+            )
+
+        return self._sm_client
+
+    @property
+    def secret(self):
+        if not self._cache.get("secret"):
+            arn = self.conf.get("secret_arn")
+            self.logger.info("attempting to get secret %s", arn)
+
+            response = self.sm_client.get_secret_value(SecretId=arn)
+            # get_secret_value response syntax:
+            #  {
+            #    "ARN": "string",
+            #    "Name": "string",
+            #    "VersionId": "string",
+            #    "SecretBinary": b"bytes",
+            #    "SecretString": "string",
+            #    "VersionStages": [
+            #        "string",
+            #    ],
+            #    "CreatedDate": datetime(2015, 1, 1)
+            #  }
+            #
+            # We're interested in SecretString and expect it to be a JSON string
+            # containing the "cookie_key" key.
+            if "SecretString" in response:
+                secret = response["SecretString"]
+                self.logger.info("secret string %s", repr(secret)[0:50])
```

Comment:
This one really shouldn't be info I think.  I believe I wrote this originally?  It was to help me understand what SM was returning during development, but given this is an actual secret I don't think we should permanently be logging it, even if it's just the first 50 chars.  Could you please drop it or make it debug?

Reply:
Thanks for pointing this out. Improving logging all around now.